homeassistant/components/light/services.yaml

```
@@ -631,9 +577,9 @@ toggle:
     effect:
       name: Effect
       description: Light effect.
-      example: random
-      values:
-        - colorloop
-        - random
       selector:
-        text:
+        select:
+          options:
+            - colorloop
+            - random
+            - white
```

Effects can be anything
```suggestion
        text:
```